brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/LanguageCombination.kt

```diff
@@ -1,27 +1,54 @@
 package no.nav.pensjon.brev.template
 
+import java.util.Objects
+
 internal sealed class LanguageCombination {
 
-    data class Single<Lang : Language>(val first: Lang) : LanguageCombination(), LanguageSupport.Single<Lang>, StableHash by StableHash.of(first) {
+    class Single<Lang : Language>(val first: Lang) : LanguageCombination(), LanguageSupport.Single<Lang>, StableHash by StableHash.of(first) {
```

Comment:
Hva er grunnen til at disse klassene trenger å være public?

Reply:
Brukt som returverdi frå languages-funksjonen i Utils, som igjen er brukt i createTemplate-kallet i malane. Ser inga opplagt betre løysing for det.

Reply:
Men kan fint ha intern konstruktør, da.